src/main/java/org/example/intro/controller/BookController.java

```diff
@@ -23,7 +22,8 @@ public List<BookDto> findAll() {
     }
 
     @PostMapping
-    public BookDto create(@RequestBody CreateBookDto requestDto) {
+    @ResponseStatus(HttpStatus.CREATED)
+    public BookDto create(@RequestBody @Valid CreateBookDto requestDto) {
```

Comment:
Validate dto in search endpoint

Reply:
Unfortunately i haven't figured out how to validate parameters here. I saw that we can validate incoming data in the request body. But this is GET query. 
The only thing i had found - is validating the type of request parameter and added additional method to the "CustomGlobalExceptionHandler" class.
If i can improve it - could you please help me with that? what certainly to make here? any resources where i would be able to take that information.
Thank you!

Reply:
My mistake, don't need to validate